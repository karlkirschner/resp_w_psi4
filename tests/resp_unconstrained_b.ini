# read in saved esp and grid files
# If you know how these files were created, you should fill in:
#   - vdw_scale_factors
#   - vdw_radii, 
#   - method_esp
#   - basis_esp

[molecules]
input_files: acetic_acid.xyz
    
[vdw.surface.options]
vdw_scale_factors : 1.4, 1.6, 1.8, 2.0
vdw_point_density : 1.0
esp               : acetic_acid_esp.dat
grid              : acetic_acid_grid.dat
vdw_radii         : H=1.20,
                    C=1.50,
                    O=1.40

[hyperbolic.restraint.options]
weight            : 1.0
restraint         : True
resp_a            : 0.0005
resp_b            : 0.1
ihfree            : True
toler             : 1e-5
max_it            : 25 

[constraints]
constraint_charge : None
equivalent_groups : None

[qm.options]
method_esp    : None
basis_esp     : None
formal_charge : 0
multiplicity  : 1