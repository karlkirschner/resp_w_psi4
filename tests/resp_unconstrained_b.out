Electrostatic potential parameters
    van der Waals radii (Angstrom):
                                      C = 1.500
                                      H = 1.200
                                      O = 1.400
    VDW scale factors:                1.4 1.6 1.8 2.0
    VDW point density:                1.0
    ESP method:                       None
    ESP basis set:                    None

Grid information
    Quantum ESP File(s):
                                      acetic_acid_esp.dat

    Grid Points File(s) (# of points):
                                      acetic_acid_grid.dat (534)

Constraints
    Charge constraints: None

    Equivalent charges on atoms: None

Restraint
    ihfree:                           True
    resp_a:                           0.0005
    resp_b:                           0.1000

Fit
    Electrostatic Potential Charges:
        Center  Symbol        esp         resp        
          1        C      -0.47448278 -0.29504688
          2        H       0.15053453  0.10713421
          3        H       0.15053453  0.10713421
          4        H       0.13074757  0.08483229
          5        C       0.88861005  0.80432821
          6        O      -0.67611995 -0.66146569
          7        H       0.45167235  0.45334413
          8        O      -0.62149630 -0.60026048

        Total Charge:      0.00000000 -0.00000000
